graphgym/main.py

```diff
@@ -40,14 +40,15 @@ def new_scheduler_config(cfg):
     load_cfg(cfg, args)
     # Set Pytorch environment
     torch.set_num_threads(cfg.num_threads)
-    dump_cfg(cfg)
-    set_printing()
+    dump_cfg(cfg, args.cfg_file)
```

Comment:
`dump_cfg` does not need to be changed. The purpose is to save a copy of cfg to the results folder